### DISCUSSION TITLE
wireguard出站如何处理目标为域名的连接？

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#1507] [Q&A]
wireguard原生是不支持域名为访问目标的，也就不可以直接把域名交给wireguard的outbound，那么它是怎么对待域名目标的？是通过内置DNS解析？还是可以像freedom出口那样设置domainStrategy？文档里面没有提到这个问题，希望后续可以补充说明一下。

**Comment by kbr3NwSZ6kFq25** [comment]
实测会用内置dns解析
